storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -27,6 +27,11 @@ In future {product-title} versions, volumes provisioned using existing in-tree p
 After full migration, in-tree plug-ins will eventually be removed in future versions of {product-title}.
 ====
 
+[NOTE]
+====
+The vSphere CSI Driver supports dynamic and static provisioning. When using static provisioning in the PV specifications, do not use the key `storage.kubernetes.io/csiProvisionerIdentity` in `csi.volumeAttributes` because this key indicates dynamically provisioned PVs.
```

Comment:
Change to: "vSphere CSI Driver Operator"

Reply:
> Change to: "vSphere CSI Driver Operator"

Hold. Volumes dynamic and static provisioning is supported by driver not operator. No need to change.Thanks!

Reply:
no change here

Reply:
@lpettyjo please merge